### PR TITLE
[BACKLOG-36861] - Page Zoom (AA) - Manage Data Source Dialog

### PR DESCRIPTION
**Pull request by soagarwal1**
@pentaho/wcag Please Review

This PR is a part of 
https://github.com/pentaho/data-access/pull/1211
https://github.com/pentaho/pentaho-commons-xul/pull/252

### CODE REVIEW

**Comment by buildguy** [comment]
### :warning: Build finished in 6m 16s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 1 test(s) failed:

<details><summary><b>org.pentaho.gwt.widgets.client.table.BaseTableTest.testShowMessage</b> (click to expand)</summary><p>

```

Argument(s) are different! Wanted:
baseTable.createTable(
    <any>,
    <any>,
    <any>,
    FIXED_WIDTH,
    <any>
);
-> at org.pentaho.gwt.widgets.client.table.BaseTableTest.testShowMessage(BaseTableTest.java:73)
Actual invocation has different arguments:
baseTable.createTable(
    ["&nbsp;"],
    null,
    [[Ljava.lang.String;@6060f163],
    FILL_WIDTH,
    null
);
-> at org.pentaho.gwt.widgets.client.table.BaseTableTest.testShowMessage(BaseTableTest.java:70)

```

</p></details>

  
Tests run: **428**, Failures: **1**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-941/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 7s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **428**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-941/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — CHANGES_REQUESTED** [review]

File: widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/ElementUtils.java [hunk 1]
```diff
@@ -446,4 +446,8 @@ public static native void scrollVerticallyIntoView( Element elem ) /*-{
   public static native boolean isEventDefaultPrevented( NativeEvent evt ) /*-{
     return evt.defaultPrevented;
   }-*/;
+
+  public static native void log( String text ) /*-{
+      $wnd.console.log( text );
+  }-*/;
```
[the remark this answers is not in the document]
Comment (soagarwal1) [reply widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/ElementUtils.java]: Done

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **428**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-941/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by varuntangirala — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 37s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **428**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-941/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**